frontend/src/app/specification-overview/specification-overview.component.css

```diff
@@ -1,3 +1,16 @@
 button {
   margin-right: 5px;
 }
+
+#title-column {
+  width: 60%;
+}
+
+#version-column {
+  width: 20%;
+}
+
+#actions-column {
+  width: 20%;
```

Comment:
Can we tell it to make this as small as possible, and use the rest for the other columns?

Reply:
I'm a little uneasy about mixing up pixel and percent measurements. (which I assume was the proposition, as the oi buttons are of fixed size?) After searching I couldn't find a clear consensus as to whether it was safe. Also, the button to edit permissions on a specification conditionally appears based on whether the logged in user has those access rights, so I did not see this column as having an objective minimum width.

Reply:
I'd probably have tried to set it to some ridiculous amount like `5%` :)